app/controllers/admin/content_controller.rb

```diff
@@ -113,6 +113,20 @@ def autosave
     render :text => nil
   end
 
+  def merge
+    @article = Article.find(params["id"])
+    merge_article = Article.find(params[:merge_with])
```

Comment:
It might be good if this logic was in an Article model method instead of directly in the controller